first_cycling_api/race/endpoints.py

```diff
@@ -85,14 +86,14 @@ def _parse_soup(self):
 		self._get_sidebar_information()
 
 	def _get_results_table(self):
-		results_table = self.soup.find('table', {'class': 'sortTabell tablesorter'})
+		results_table = self.soup.find('table', {'class': 'sortTabell'})
 		if not results_table:
-			results_table = self.soup.find('table', {'class': 'sortTabell2 tablesorter'})
+			results_table = self.soup.find('table', {'class': 'sortTabell2'})
```

Comment:
The magic lies in these lines, which were previously excluding the tables since FirstCycling changed their class names.